aten/src/ATen/native/cuda/UnaryOpsKernel.cu

```diff
@@ -9,6 +9,27 @@
 
 namespace at { namespace native {
 
+template<typename T>
+T __host__ __device__ abs_impl(T v) {
+  return std::abs(v);
+}
+template<>
+uint8_t __host__ __device__ abs_impl(uint8_t v) {
+  return v;
+}
+template<>
+bool __host__ __device__ abs_impl(bool v) {
+  return v;
+}
+
+void abs_kernel_cuda(TensorIterator& iter) {
+  AT_DISPATCH_ALL_TYPES_AND(ScalarType::Half, iter.dtype(), "abs_cuda", [&]() {
```

Comment:
`bool` is not counted in. Use

    AT_DISPATCH_ALL_TYPES_AND2(ScalarType::Half, ScalarType::Bool, ...